Dockerfile.build

```diff
@@ -2,13 +2,16 @@
 # --- stage 1 ---
 FROM registry.ci.openshift.org/openshift/release:golang-1.16 as builder
 
+ENV GO111MODULE=on
```

Comment:
```suggestion
```
can be removed as this is the default behaviour anyway with Go 1.16 :)

Reply:
done